ws-slave.js

```diff
@@ -766,3 +726,15 @@ noble.on('discover', function (peripheral) {
   //console.log("noble thinks we discovered ", peripheral.advertisement.localName);
   handleDiscoveredPeripheral(peripheral);
 });
+
+noble.on('scanStop', function () {
+    //console.log("Scanning stopped");
```

Comment:
Could maybe put a short comment in talking about the behaviour you discovered and why this is here.

But also: WOOOOOO YOU FIXED IT.